frontend/src/pages/wifiPage/connectDialog/ConnectDialog.tsx

```diff
@@ -105,8 +105,8 @@ export default ({
           <span className={styles.error}>
             {disconnectedFromAp ? (
               <>
-                Your computer has disconnected from the wifi network
-                'pi-top'. Please reconnect to it.
```

Comment:
I think if the ssid above is using green instead of inverted commas then this should as well. Or we could say 'disconnected from the pi-top's WiFi hotspot' instead?